experiments/ext_api/word_count_demo/s3_sender.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import print_function
+
+import sys
+import threading
+
+import boto3
+from word_counts import CountStream, parse_count_stream_addr
+
+count_stream_addr = parse_count_stream_addr(sys.argv)
+extension = CountStream(*count_stream_addr).extension()
+s3 = boto3.client('s3')
```

Comment:
I think we should be clearer on how we get authentication parameters into the library here. I've not been able to test some of these myself but for end users, mixing some CLI and some ENV stuff is a bit ad-hoc. Obv. passing secrets on the CLI is not ideal but it is consistent. For now, I'd prefer that we parse these out and then discuss how we want to handle secret parameters in Wallaroo so we can keep everything consistent.